flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/factory/KubernetesJobManagerFactoryTest.java

```diff
@@ -468,4 +468,37 @@ void testSetJobManagerDeploymentReplicas() throws Exception {
         assertThat(kubernetesJobManagerSpecification.getDeployment().getSpec().getReplicas())
                 .isEqualTo(JOBMANAGER_REPLICAS);
     }
+
+    @Test
+    void testDNSPolicyDefaultValue() throws Exception {
```

Comment:
I think this test is not very necessary. If you want to keep it, then`InitJobManagerDecoratorTest` might be a better place.